css/output.css

```diff
@@ -607,10 +599,22 @@ video {
   margin-bottom: 2.5rem;
 }
 
+.mb-12 {
```

Comment:
@Klarala13 did you add these or was it automatic? We're using a css framework called Tailwind CSS that has these by default, you then just add them as css classes.

Reply:
It was automatic, I use tailwind but when I did the install, they appeared. Can delete them if you think.

Reply:
ok cool, no no it's cool that was what I wanted to check. Thanks :)